src/components/BuildConfigDetailPage/BuilConfigRevisionPage.tsx

```diff
@@ -12,10 +12,28 @@ export const BuildConfigRevisionPage = () => {
 
   const serviceContainerBuildConfigRevision = useServiceContainer(buildConfigApi.getRevision);
   const serviceContainerBuildConfigRevisionRunner = serviceContainerBuildConfigRevision.run;
+  const serviceContainerBuildConfig = useServiceContainer(buildConfigApi.getBuildConfig);
+  const serviceContainerBuildConfigRunner = serviceContainerBuildConfig.run;
+
+  const isCurrentRevision = () => {
+    if (!serviceContainerBuildConfig.data || !serviceContainerBuildConfigRevision.data) {
+      return false;
+    }
+    return serviceContainerBuildConfig.data?.modificationTime === serviceContainerBuildConfigRevision.data?.modificationTime;
```

Comment:
I have two points:

1) Would it be possible to use ID based comparison instead of `modificationTime` based? We have list of all revisions already loaded and we know the last one is the current one.

2) The list of all revisions is already loaded on `BuildConfigRevisionPages` and I think the last revision ID could be shared via `useOutletContext` ? So that no other http request is performed. Your current solution is loading new Build Config for each revision, let's try to avoid additional HTTP requests.